projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -639,6 +659,35 @@ public function edit_dsp_generic( $path, $req, $args = array() ) {
 		);
 	}
 
+	/**
+	 * Will check the posts for prices and add them to the posts array
+	 *
+	 * @param WP_REST_Request $posts The posts object.
+	 * @return array|WP_Error
+	 */
+	protected function add_prices_in_posts( $posts ) {
+
+		if ( ! function_exists( 'wc_get_product' ) || ! function_exists( 'wc_price' ) ) {
+			return $posts;
+		}
+
+		foreach ( $posts as $key => $post ) {
+			$product = wc_get_product( $post['ID'] );
+			if ( $product !== false ) {
+				$posts[ $key ]['price'] = html_entity_decode(
+					wp_strip_all_tags(
+						wc_price( $product->get_price() )
+					),
+					ENT_QUOTES
+				);
```

Comment:
I take it there is no simple function to retrieve a formatted and escaped product price?

Reply:
@jeherve  sorry for the delay in responding... we were in a division meetup.

i did some research and asked around, I changed the code (even though it is more lines of code) i think it is better to not use entity_decode and strip tags

@sbarbosa please let me know what you think as well